src/main/java/ladder/model/RandomLineCreateDecider.java

```diff
@@ -0,0 +1,15 @@
+package ladder.model;
+
+import java.security.SecureRandom;
+
+public class RandomLineCreateDecider implements LineCreateDecider {
+
+    private static final int MAX_RANDOM_INT = 2;
+    private static final SecureRandom secureRandom = new SecureRandom();
```

Comment:
컨벤션상 아래와 같이 명명하는 것이 좋아 보입니다 :)
```suggestion
    private static final SecureRandom SECURE_RANDOM = new SecureRandom();
```

Reply:
상수인것을 미처 고려하지 못하였네요. 수정하였습니다!